source/_docs/automation/templating.markdown

```diff
@@ -11,6 +11,10 @@ The template variable `this` is also available when evaluating any `trigger_vari
 
 `this` is a state object. [State Objects](/docs/configuration/state_object) provides a comprehensive description for the properties of `this` and `this.attributes`.
 
+## Available context Data
+
+Each of `this`, `to_state` and `from_state` have a context object, which includes information such as the user that caused the change. [Home Assistant Context](https://data.home-assistant.io/docs/context/t) provides more detail on contexts.
```

Comment:
Besides the link is broken, I would not send users out of the documentation. If this is relevant for end-user documentation, it should be added to the end-user documentation.

Reply:
That makes sense, happy to keep everything within the main docs site. I wasn't sure it warranted its own page - would it be a better fit on the State Objects page or are contexts not strictly part of those?

Reply:
Though call, the context object comes as a part of the state object, so that seems like a good fit to document on the states object page (to keep it more centralized). A separate page would be cool too, however, I think there is not much to talk about. I guess in general the `user_id` is the only thing of interest in that case.

What do you think?

Reply:
Yeah, I think some extra detail on the State Objects page probably fits better, given there aren't many details for the context object itself. I'll put something together in a bit and update the PR.

Reply:
Awesome! 👍

Reply:
Is that better? I've left a note in the state object section of the automation templating page to signpost users there, and added the detail itself over on the state object page.